Clippy error

cargo clippy:

```
error[E0433]: failed to resolve: could not find `web_sys` in `leptos`
   --> src/app.rs:35:5
    |
35  | /     leptos_fluent! {{
36  | |         locales: "./locales",
37  | |         translations: TRANSLATIONS,
38  | |         check_translations: "./src/**/*.rs",
...   |
46  | |         set_language_to_cookie: true,
47  | |     }};
    | |______^ could not find `web_sys` in `leptos`
    |
note: found an item that was configured out
   --> /Users/vasily/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptos-0.6.12/src/lib.rs:239:9
    |
239 | pub use web_sys; // used in islands
    |         ^^^^^^^
    = note: this error originates in the macro `leptos_fluent` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0433`.
error: could not compile `metamenu` (lib) due to 1 previous error
```

Though it builds fine.

--- a/leptos-fluent-macros/Cargo.toml
+++ b/leptos-fluent-macros/Cargo.toml
@@ -2,7 +2,7 @@
 name = "leptos-fluent-macros"
 description = "Macros for leptos-fluent"
 edition.workspace = true
-version = "0.1.2"
+version = "0.1.3"
 license = "MIT"
 documentation.workspace = true
 repository.workspace = true

--- a/leptos-fluent-macros/src/lib.rs
+++ b/leptos-fluent-macros/src/lib.rs
@@ -253,7 +253,7 @@ pub fn leptos_fluent(
                 _ = ::leptos::document()
                     .document_element()
                     .unwrap()
-                    .unchecked_into::<::leptos::web_sys::HtmlElement>()
+                    .unchecked_into::<::leptos_fluent::web_sys::HtmlElement>()
                     .set_attribute(
                         "lang",
                         &::leptos_fluent::expect_i18n().language.get().id.to_string()
@@ -293,7 +293,7 @@ pub fn leptos_fluent(
                 _ = ::leptos::document()
                     .document_element()
                     .unwrap()
-                    .unchecked_into::<::leptos::web_sys::HtmlElement>()
+                    .unchecked_into::<::leptos_fluent::web_sys::HtmlElement>()
                     .set_attribute(
                         "dir",
                         ::leptos_fluent::expect_i18n().language.get().dir.as_str(),

--- a/leptos-fluent/Cargo.toml
+++ b/leptos-fluent/Cargo.toml
@@ -2,7 +2,7 @@
 name = "leptos-fluent"
 description = "Fluent framework for internationalization of Leptos applications"
 edition.workspace = true
-version = "0.1.2"
+version = "0.1.3"
 license = "MIT"
 documentation.workspace = true
 repository.workspace = true

--- a/leptos-fluent/src/lib.rs
+++ b/leptos-fluent/src/lib.rs
@@ -239,6 +239,9 @@ pub mod localstorage;
 #[doc(hidden)]
 pub mod url;
 
+#[doc(hidden)]
+pub use web_sys;
+
 use core::hash::{Hash, Hasher};
 use core::str::FromStr;
 use fluent_templates::{

